src/controller/ShopController.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const Router = express.Router
+const Shop = require('../models/shopSchema')
+const ShopController = new Router()
+
+// LIST ALL SHOPS - shops/list
+ShopController.get('/list', (req, res, next) => {
+    console.log('List all shops')
+    
+    res.send('yay')
+})
+
+
+
+// CREATE A SHOP - shops/create
+ShopController.post('/create', (req, res, next) => {
+    const Topshop = new Shop({
+        name: "Topshop",
+        type: "Clothes", 
+        location: {
+            postcode: "SP10 3NG",
+            town: "Andover"
+        }
+    })
+    res.send(Topshop)
```

Comment:
You typically do `res.send` at the end. Why do you think that wold be?

Reply:
I would put this at the end as I see this is a form of return statement, it should be at the end so that any changes to that variable are sent in its final form. However, it doesn't actually return and you are able to make changes to the variable after it